iceoryx_hoofs/platform/win/include/iceoryx_hoofs/platform/platform_correction.hpp

```diff
@@ -28,6 +28,8 @@
 #define __PRETTY_FUNCTION__ __FUNCSIG__
 #endif
 
+#pragma pointers_to_members(full_generality)
```

Comment:
The caution note here https://docs.microsoft.com/en-us/cpp/preprocessor/pointers-to-members?redirectedfrom=MSDN&view=msvc-170 makes me nervous but I guess we need it ...

Reply:
Yes, msvc is a mess